client/README.md

```diff
@@ -227,113 +227,19 @@ You can also specify multiple namespaces when using the hook:
 
 ## Android App
 
-Short version to build an Android apk (if everything is already set up correctly):
+Please see android folder in server and android folder in react for further details, including prerequisite setup steps required for commands below
 
-```bash
-# Compile the remote server and copy libs over:
```

Comment:
Relocated to packages/android/README.md (and there are instruction on how to build server in server/android/README.md, but that hasn't change from previous PR)